doc/developers/contributing.rst

```diff
@@ -456,6 +468,11 @@ There are two other tags to help new contributors:
     Might need some knowledge of machine learning or the package,
     but is still approachable for someone new to the project.
 
+:New Contributor:
+    This tag marks an issue which is currently lacking a contributor.
```

Comment:
Again I would mention that this tag might not be applied to all issues/ PRs that need contributors.